### ISSUE TITLE
Paymill bridge css3 JS

### ISSUE THREAD

**Issue by tillhess** [paymill/paymill-magento#37]
Is it really necessary to include the JS on every single page?

**Comment by peterjaap** [comment]
No, you can just edit the layout XML file to only have it shown on the checkout.

**Comment by ESYdesignmarketing** [comment]
Actually we have the same problem. So what would be the correct way of doing it?
Thanks for your help.
